### PR TITLE
Add a flag to disable taking screenshots

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

File: src/screenshots/index.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ import getCommonPath from '../utils/get-common-path';
 const SCREENSHOT_EXTENSION = 'png';
 
 export default class Screenshots {
-    constructor ({ path, pathPattern, fullPage } = {}, { screenshotPathPattern }) {
-        this.enabled            = !!path;
+    constructor ({ enabled, path, pathPattern, fullPage }) {
+        this.enabled            = enabled;
         this.screenshotsPath    = path;
-        this.screenshotsPattern = pathPattern || screenshotPathPattern;
+        this.screenshotsPattern = pathPattern;
```
Comment (AlexKamaev) [inline src/screenshots/index.js]: after this refactoring we do not need `The `screenshots` option integration` test, which fails on CI now

File: src/notifications/warning-message.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export default {
+    screenshotsDisabled:                     'The screenshot functionality is disabled. Remove the "--disable-screenshots" command line option or the "disableScreenshots" runner and configuration option to enable it.',
```
Comment (VasilyStrelyaev) [inline src/notifications/warning-message.js]: Checked:
```
Screenshots are disabled. To take screenshots, remove the "--disable-screenshots" command line flag or set the "disableScreenshots" option to "false" in the API or configuration file.
```

File: src/cli/argument-parser.ts [hunk 3]
```diff
@@ -124,13 +124,14 @@ export default class CLIArgumentParser {
...
             .option('--video-encoding-options <option=value[,...]>', 'specify encoding options')
-            .option('--disable-page-reloads', 'disable page reloads between tests')
             .option('--dev', 'enables mechanisms to log and diagnose errors')
             .option('--qr-code', 'outputs QR-code that repeats URLs used to connect the remote browsers')
             .option('--sf, --stop-on-first-fail', 'stop an entire test run if any test fails')
             .option('--ts-config-path <path>', 'use a custom TypeScript configuration file and specify its location')
             .option('--cs, --client-scripts <paths>', 'inject scripts into tested pages', this._parseList, [])
             .option('--disable-page-caching', 'disable page caching during test execution')
+            .option('--disable-page-reloads', 'disable page reloads between tests')
+            .option('--disable-screenshots', 'disable taking screenshots')
```
Comment (VasilyStrelyaev) [inline src/cli/argument-parser.ts]: I would say (though I haven't checked it, but `disable taking` won't pass anyway)

```sj
            .option('--disable-screenshots', 'disable screenshots')
```

Comment (VasilyStrelyaev) [inline src/cli/argument-parser.ts]: Please also change this to just:

```js
            .option('--disable-screenshots', 'disable screenshots')
```

Comment (VasilyStrelyaev) [reply src/cli/argument-parser.ts]: And also: `set the amount of time` -> `specify the time`

```js
            .option('--selector-timeout <ms>', 'specify the time within which selectors make attempts to obtain a node to be returned')
            .option('--assertion-timeout <ms>', 'specify the time within which assertion should pass')
            .option('--page-load-timeout <ms>', 'specify the time within which TestCafe waits for the `window.load` event to fire on page load before proceeding to the next test action')
```

Comment (VasilyStrelyaev) [reply src/cli/argument-parser.ts]: And... the description for `-e` is printed on the same line:

image

Comment (VasilyStrelyaev) [reply src/cli/argument-parser.ts]: And an extra space in the `--video` description
```js
            .option('--video <path>', 'record videos of test runs')
```

**Pull request merged**